Initium-Odp/src/com/universeprojects/miniup/server/model/GridObject.java

```diff
@@ -29,6 +29,94 @@ public GridObject(String key, String fileName, String name, int xGridCoord, int
 		this.height = height;
 	}
 
+	public String getKey() {
```

Comment:
Same comment as the GridCell, storing everything in a map or JSONObject I think would be easier long-term.